README.md

```diff
@@ -10,47 +10,99 @@ Requires [NodeJS](https://nodejs.org/en/) 12+:
 
 ```console
 $ npm install
-$ npm link
 ```
 
-## Usage
-
-### API
+If you want to have `jsv` available _globally_ in your path, you can use `npm link`.
 
-To convert [`input.json`](test/fixtures/input.json) into [`output.md`](test/fixtures/output.md), we need to read the content of the input file pass it to the `toMarkDown` function. For example, the following function would return the contents of `output.md` when called with the contents of `input.json`:
-
-```javascript
-async function convert(path) {
-  const input = await fs.promises.readFile(path, "utf8");
-  return toMarkDown(input);
-}
-```
+## Usage
 
-### CLI (to be implemented)
+### CLI
 
 ```console
 $ jsv --help
-Usage: jsv [options] <URL> <OUTPUT>
+Usage: jsv [options] [<json>]
 
 jsv (JSON Scheme Viewer)
 
 JSON Schema viewer is a lightweight javascript library and tool that turns JSON
 schemas into a elegant human readable documents.
 
-It expects URL to be a location of a JSON scheme file, and it decides the
-format of the output based on the file extension of OUTPUT (which is created
-as a result) or the format provided via the --output option.
+It expects a JSON Schema from stdin and outputs to stdout its version for
```

Comment:
I don't line wrap at 80 characters in READMEs usually.

Reply:
This is the text generated by the CLI and I tend to just copy & paste it. Commander wraps these lines, maybe. I tend to leave it as it is (after pasting) for a more realistic look & feel in the docs. What do you think?